lib/ParamsCalculator/aif.py

```diff
@@ -1,18 +1,495 @@
+import numpy as np
+from numpy.lib.function_base import trim_zeros
+from scipy.integrate import trapz
 import os
+import math
 import torch
 import logging
-import numpy as np
 from scipy import signal
-import SimpleITK as sitk
+import itk
+import sys
 from builtins import object
+from scipy import linalg
+from scipy.optimize import least_squares
+from scipy.signal import convolve
+import matplotlib.pyplot as plt
+from scipy.ndimage import label
+from scipy.cluster.hierarchy import linkage, fcluster
+from scipy.spatial.distance import pdist
+from scipy.optimize import least_squares
+
+def DSC_aif(mask, conc, bolus, size, config, device, vessels):
+    nSlice = 69
```

Comment:
@aylward @Caffetaria Could you comment on the motivation for using this hardcoded slice value?

My guess is that this always extracts time point 68 from the 4D image. But, our data only appears to have 16 time points, leading to an error here.

Reply:
> This is probably the z-slice from the 3D volumes that comprise the 4D images. This number should come from an atlas-registration process, that she didn't have time to implement.

Makes sense, and that's fine. We can move forward with the value in this PR, but for subsequent testing I'm unclear on whether we have an input data set that works with this parameter.

If this is an arbitrary z-slice index, would it be equally valid to select a different z-slice index that fits with our test data so that the application does not fail at this point? I'm unfamiliar with the consequences of which z-slice we select here.

Reply:
We had two sets of testing data: high res and low res.

The high res 4D volume should be 16 time points with 376x376x170 volumes.   The nSlice value probably corresponds to the high res testing data

If you're using the low res 4D volume (size 16 time points with 138x138x18 volumes).  The nSlice value should be 7 for the low res testing data

Hope this helps.

Reply:
> We had two sets of testing data: high res and low res.

This is a piece I was missing. I see the [high-res mask](https://data.kitware.com/#item/64da58a81c6956f5031e433f) but I'm not seeing a corresponding input volume in our [sample data](https://data.kitware.com/#folder/64da58971c6956f5031e433d). The high-res mask has size 170x376x376, whereas `CTP04-4D_reg.nii.gz` has size 16x18x138x138.

Could you please point me towards the high-res 4D input volume?